debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/antlr/listener/CreateTableParserListener.java

```diff
@@ -44,6 +48,10 @@ public CreateTableParserListener(MySqlAntlrDdlParser parser, List<ParseTreeListe
     @Override
     public void enterColumnCreateTable(MySqlParser.ColumnCreateTableContext ctx) {
         TableId tableId = parser.parseQualifiedTableId(ctx.tableName().fullId());
+        if (!parser.getTableFilter().isIncluded(tableId)) {
+            LOG.debug("Ignoring CREATE TABLE statement for non-captured table {}", tableId);
+            return;
+        }
```

Comment:
- This check prevents the `parser.databaseTables().editOrCreateTable(tableId)` call on tables that are exluded by the configuration. 
- Later on `AlterTableParserListener` [won't instantiate](https://github.com/debezium/debezium/blob/48ca58a609610e0bd38297738aab911da055563f/debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/antlr/listener/AlterTableParserListener.java#L58-L62) a `tableEditor` since this table is not represented in the parser internals. 
- By the time a "modify column" DDL arrives `tableEditor` is going to be `null`. This is going to prevent [enterAlterByModifyColumn](https://github.com/debezium/debezium/blob/48ca58a609610e0bd38297738aab911da055563f/debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/antlr/listener/AlterTableParserListener.java#L203) from executing the main body of the method. 

This implements [the todo](https://github.com/debezium/debezium/blob/48ca58a609610e0bd38297738aab911da055563f/debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseSchema.java#L78) from `RelationalDatabaseSchema.tableIds`.